src/openrct2-ui/input/MouseInput.cpp

```diff
@@ -484,13 +484,11 @@ static void input_window_resize_continue(rct_window* w, ScreenCoordsXY screenCoo
 {
     if (screenCoords.y < (int32_t)context_get_height() - 2)
     {
-        int32_t dx, dy, targetWidth, targetHeight;
-        dx = screenCoords.x - gInputDragLast.x;
-        dy = screenCoords.y - gInputDragLast.y;
-        targetWidth = _originalWindowWidth + dx;
-        targetHeight = _originalWindowHeight + dy;
+        auto differentialCoords = ScreenCoordsXY{ screenCoords.x - gInputDragLast.x, screenCoords.y - gInputDragLast.y };
```

Comment:
Any reason why this cant be `screenCoords - gInputDragLast`

Reply:
There wasn't, I was just not my best self 👶